src/TLuaInterpreter.cpp

```diff
@@ -2477,6 +2478,32 @@ int TLuaInterpreter::closeMudlet(lua_State* L)
     return 0;
 }
 
+int TLuaInterpreter::saveProfile(lua_State* L)
+{
+    Host* pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if (!pHost) {
+        lua_pushstring(L, QLatin1String("saveProfile: NULL Host pointer - something is wrong!").data());
+        return lua_error(L);
+    }
+
+    QString saveToDir;
```

Comment:
it __is__ a pathFileName not a pathName i.e. the complete name and path to use for the save file name, although including the ".dat" extension is optional but might confuse dumb file-systems that place too much reliance on them (looking at you, Micro$oft)...